integration-cli/docker_cli_push_test.go

```diff
@@ -64,9 +59,8 @@ func (s *DockerRegistrySuite) TestPushMultipleTags(c *check.C) {
 
 	// Ensure layer list is equivalent for repoTag1 and repoTag2
 	out1, _ := dockerCmd(c, "pull", repoTag1)
-	if strings.Contains(out1, "Tag t1 not found") {
-		c.Fatalf("Unable to pull pushed image: %s", out1)
-	}
+	c.Assert(out1, check.Not(checker.Contains), "Tag t1 not found", check.Commentf("Unable to pull pushed image: %s", out1))
```

Comment:
I wonder if this check is needed at all. I think `docker pull` will return an error (at least `exit 1`) if failing with `docker pull myimage:nonexistenttag`, so this might be handled already by `dockerCmd`.. Something to try out :smile_cat: